lib/deny-list.js

```diff
@@ -0,0 +1,3 @@
+const denyList = [];
```

Comment:
[Question] Do we know what was previously served, and should that data go here? Is it https://github.com/yeoman/yeoman-generator-list/blob/3536bf0c8f0ba47b28a1f00bf8850982108c16b4/src/blacklist.js?

Reply:
Good question.